controlpanel/frontend/static/javascripts/modules/tool-deploy.js

```diff
@@ -1,5 +1,5 @@
 moj.Modules.tool_deploy = {
-  class: ".deploy-tool",
+  class: ".tool-deploy-form",
```

Comment:
[What I did](https://github.com/ministryofjustice/analytics-platform-control-panel/commit/4f5df1955ac37a1b103152b7ec027df965324d8f) in my branch was to change the CSS class to `form.background-submit` as I wanted pretty much the same behaviour for the "Upgrade" button.

Don't know if that makes sense but happy for you to get that if it does.

Reply:
Yeah, I like what you did, although I might just add `background-submit` rather than replace `tool-deploy-form`. But I'll let you submit that change in your own PR :)